features/logging/deploy_logging_operators_via_console.feature

```diff
@@ -0,0 +1,77 @@
+Feature: Deploy logging operators via console
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22558
+  @admin
+  @destructive
+  Scenario: Deploy cluster-logging operator via web console.
+    # clear the logging related resources
+    Given logging service is removed successfully
+    Given I switch to cluster admin pseudo user
+    Given logging channel name is stored in the :logging_channel clipboard
+    Given I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/deploy_clo_via_olm/01_clo_ns.yaml |
+    Then the step should succeed
+    Given "cluster-logging" packagemanifest's operator source name is stored in the :clo_opsrc clipboard
+    Given I switch to the first user
+    Given the first user is cluster-admin
+    Given I open admin console in a browser
+    # subscribe cluster-logging-operator
+    When I perform the :goto_operator_subscription_page web action with:
+      | package_name     | cluster-logging     |
+      | catalog_name     | <%= cb.clo_opsrc %> |
+      | target_namespace | openshift-logging   |
+    Then the step should succeed
+    Given I register clean-up steps:
+    """
+    Given logging service is removed successfully
+    Then the step should succeed
+    """
```

Comment:
Looks like clean-up step is not immediately before/after step creating the thing to be removed. This makes possible to miss clean-up if previous steps failed.